pkg/iterator/emitters.go

```diff
@@ -76,7 +79,82 @@ func (e *TableEmitter) SnapshotMetadataIteratorRecord(recordNumber int, metadata
 		fmt.Fprintf(e.Writer, tableRowFmt, recordNumber, metadata.VolumeCapacityBytes, bmt, bmd.ByteOffset, bmd.SizeBytes)
 	}
 
-	return true
+	return nil
+}
+
+func (e *TableEmitter) SnapshotMetadataIteratorDone(_ int) error {
+	return nil
+}
+
+// VerifierEmitter formats the metadata as a table.
```

Comment:
Please change the comment description!

Anything in `pkg/iterator` is part of our external interface: I don't think this functionality should be here!

It has very specific semantics which are not part of the published KEP. I suggest it be in the CLI command package as a sibling of `main.go`.